syntax = "proto3";

package agreements;

message AgreementMetadata {
  string agreement_id = 1;
  string user_uuid = 2;
  string disk = 3;
  string file_name = 4;
  string content_type = 5;
  int64  file_size = 6;
  string created_at = 7;
}
